bga_database/static/js/search.js

```diff
@@ -18,6 +18,7 @@ function makeQuerystring(params, term) {
     return $.param(params);
 }
 
+/* jshint ignore:start */
```

Comment:
When I remove the ignore directives in this code block (this start/end pair, and the line directive below), `jshint` runs without error locally.

```suggestion
```